SimpleWAWS/Code/CsmExtensions/CsmResourceGroupExtensions.cs

```diff
@@ -228,6 +216,21 @@ public static partial class CsmManager
 
                 await InitFunctionsContainer(resourceGroup);
             }
+            if (resourceGroup.SubscriptionType == SubscriptionType.Linux)
+            {
+                // If the resourceGroup is assigned, don't mess with it
+                if (!string.IsNullOrEmpty(resourceGroup.UserId)) return resourceGroup;
+
+                var createdSites = new List<Task<Site>>();
```

Comment:
You probably don't need a list for this. There is only 1 resource you're creating, right?